capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -0,0 +1,33 @@
+package com.google.univiz;
+
+/**
+ * CarnegieSizeDegree is a class that holds the enum for the range of values for a school's carnegie
+ * size degree.
+ */
+public enum CarnegieSizeDegree {
+  NOT_AVAILABLE("Degree unavailable"),
+  SIZE1("Two-year college, very small"),
```

Comment:
I remember you had another problem with trying to convert integer numbers returned from scorecard REST api into these enums. Since you're now using constrctor arguments to capture the humand-readable text for these enums, you can also add another argument and capture the numeric representation of these enums:

```java
public enum CarnegieSizeDegree {
  TWO_YEAR_COLLEGE("Two-year college, very small", 1),
   //...
  private final int scorecardNumericValue;
  private final String degree;

  CarnegieSizeDegree(String degree, int scorecardNumericValue) {
    this.degree = degree;
    this.scorecardNumericValue = scorecardNumericValue;
  }

  static CarnegieSizeDegree fromNumericValue(int scorecardNumericValue) {
    for (CarnegieSizeDegree current : values()) {
      if (current.scorecardNumericValue == scorecardNumericValue) { return current; }
    }
    throw new OutOfRangeException(
        String.format("Enum numeric value not found: %d", scorecardNumericValue));
  }
```

Reply:
should i do this as opposed to what i do in [this file](capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java) from #64?

Reply:
Either way is fine. As long as you add tests to make sure it works.